document-generator-accounts/src/test/java/uk/gov/companieshouse/document/generator/accounts/handler/accounts/AccountsHandlerImplTest.java

```diff
@@ -54,15 +59,35 @@ void testGetAbridgedAccountsDataFailureFromServiceLayer() throws ServiceExceptio
         when(accountsService.getAbridgedAccounts(anyString())).thenThrow(new ServiceException("Failure in service layer"));
 
         assertThrows(ServiceException.class, () -> accountsService.getAbridgedAccounts(anyString()));
-        assertThrows(HandlerException.class, () -> accountsHandlerImpl.getAbridgedAccountsData(transaction, ACCOUNTS_RESOURCE_LINK));
+        assertThrows(HandlerException.class, () -> accountsHandlerImpl.getAbridgedAccountsData(transaction, ACCOUNTS_RESOURCE_LINK, MIME_TYPE));
     }
 
     @Test
     @DisplayName("Tests the successful return of Abridged accounts data")
     void testGetAbridgedAccountsData() throws ServiceException, HandlerException {
         when(accountsService.getAccounts(anyString())).thenReturn(createAccountsObject());
-        when(accountsService.getAbridgedAccounts(anyString())).thenReturn(new AbridgedAccountsApi());
-        assertNotNull(accountsHandlerImpl.getAbridgedAccountsData(transaction, ACCOUNTS_RESOURCE_LINK));
+        when(accountsService.getAbridgedAccounts(anyString())).thenReturn(createCurrentPeriodAbridgedAccountObject());
+        assertNotNull(accountsHandlerImpl.getAbridgedAccountsData(transaction, ACCOUNTS_RESOURCE_LINK, MIME_TYPE));
```

Comment:
Can you verify the data returned please? This test will pass even if the response object contains no data.